app/src/main/java/com/framgia/soundcloud/service/MusicService.java

```diff
@@ -234,6 +293,30 @@ public void onBufferingUpdate(MediaPlayer mp, int percent) {
         mBufferingLevel = percent;
     }
 
+    private void runForeground() {
+        mRemoteViews =
+                new RemoteViews(getPackageName(), R.layout.layout_notification_music_service);
+        mRemoteViews.setImageViewResource(R.id.btn_next, R.drawable.ic_next);
+        mRemoteViews.setImageViewResource(R.id.btn_play, R.drawable.ic_play);
+        mRemoteViews.setImageViewResource(R.id.btn_prev, R.drawable.ic_previous);
+        mBuilder = new NotificationCompat.Builder(this).setSmallIcon(R.drawable.notification_icon)
```

Comment:
WARNING: [IconColors] Notification icons must be entirely white
 Notification icons and Action Bar icons should only white and shades of gray. See the Android Design Guide for more details. Note that the way Lint decides whether an icon is an action bar icon or a notification icon is based on the filename prefix: `ic_menu_` for action bar icons, `ic_stat_` for notification icons etc. These correspond to the naming conventions documented in http://developer.android.com/guide/practices/ui_guidelines/icon_design.html